### ISSUE TITLE
[Issue]: Trying to play a specific track from a large album fails, but hitting shuffle works

### ISSUE THREAD

**Issue by JamesSwift** [jellyfin/jellyfin#9233]
### Please describe your bug

I have this issue weather using the web client or android client. If I browse to a normal length album and hit play on a specific track everything works as expected. But if I open a large album (such as Chopin: Complete Edition) and select a track, any track, I just get a spinning circle and nothing happens. 

If however I press play on the album itself, or shuffle, it starts playing as expected.

### Jellyfin Version

10.8.0

### if other:

10.8.9

### Environment

```markdown
- OS: Ubuntu server
- Virtualization :Docker
- Clients: Web, Android
- Browser: Chrome, Edge
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy: nginx
- Base URL: none
- Networking:
- Storage: local ZFS
```

### Please attach any browser or client logs here

![image](https://user-images.githubusercontent.com/2080205/216159890-6d1e7fd7-a148-4a07-b5d1-ecfb0b8a314a.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by JamesSwift** [comment]
I can confirm this issue is still present in the latest release.

**Comment by felix920506** [comment]
How big is "large" exactly?

**Comment by JamesSwift** [comment]
One album had about 100 tracks. I'm not running jellyfin at present so I can't check exactly.

**Comment by felix920506** [comment]
I have an album with 101 FLAC tracks and it is working for me. Would need more details.

**Comment by felix920506** [comment]
Just tried again on a 258 track album, still cannot reproduce the issue

**Comment by felix920506** [comment]
Then it broke, exact same behavior, huh.

Looks like this bug isn't very reliable to reproduce

**Comment by felix920506** [comment]
Ok I think I know how to reproduce, though it's a bit random.
Successfully reproduced in Firefox, Chrome and JMP

1. Go to a large album
2. Click on a track
3. (hopefully) see error.
4. Click on another one if it plays normally

The behavior I observed is that the longer the track the more likely there are to be issues.

Additional notes:
- HTTP 414 means that the client is sending a request that is too large for the server to handle
- Doesn't happen in Jellyfin-vue or Finamp, suggesting a client issue.

**Comment by felix920506** [comment]
Upon more testing and confirmation, this issue was fixed by [jellyfin-web/#4221](https://github.com/jellyfin/jellyfin-web/pull/4221)
Closing this as it is already fixed
